spec/requests/admin/organizations_requests_spec.rb

```diff
@@ -68,7 +68,7 @@
           expect {
             post admin_organizations_path({ organization: valid_organization_params })
           }.to change(Organization, :count).by(1)
-          expect(response).to redirect_to(admin_organizations_path(organization_id: Organization.first.short_name))
+          expect(response).to redirect_to(admin_organizations_path(organization_id: 'admin'))
```

Comment:
Observing that we are using 'admin' as a hard-coded string here and several other places.  Should we gather those into a constant?

Reply:
Honestly this is kind of a hack to begin with. We really should change this so that any `admin_` paths don't get an `organization_id` at all, unless it's actually editing an organization.

Reply:
*nods* I was wondering why we had that, but it felt out of scope.  Sounds like a tech debt-level issue.

Reply:
This is true everywhere, unfortunately. We add this automatically to all URLS by default :(